Simonyi/Non-Standard-Operations/MTCS/MTHexRot/MTHex-PXI-Controller-Reboot.rst

```diff
@@ -213,16 +213,17 @@ Hard Reboot
 
     .. figure:: /Simonyi/Non-Standard-Operations/_static/mtrot-controller-pxi-reboot-1.jpeg
    
-    B.  For *MTM2Hexapod* the netbooter is required to power cycle the PXI and drives: 
-        Connect to the M2 Hex Netbooter *m2-hexapod-netbooter.cp.lsst.org* using the credential stored in the *1Password MainTel 
-        Vault* under *M2 Hexapod PXI/Drive NetBooter*.
+    B.  For *MTM2Hexapod* while in the LSST-WAP, 
+        connect to *https://pdu1-tea-as02.cp.lsst.org* using the credentials stored in the *MainTel vault* 
```

Comment:
Then you need to add a screen shot of this PDU as well. Please also include in the figure labels that these are provisional for now (as they may change !)

Reply:
<img width="1369" alt="Screenshot 2024-09-26 at 5 06 41 PM" src="https://github.com/user-attachments/assets/86b67474-3b71-4342-9f32-a4f7ce26bfad">

Reply:
Thanks! I also added the missing captions to both figures.